.scala-steward.conf

```diff
@@ -1,8 +1,3 @@
-# Only send PRs on Monday morning
-pullRequests.frequency = "0 0 ? * 1"
-
-commits.message = "build: update ${artifactName} from ${currentVersion} to ${nextVersion}"
-
 updates.ignore = [
   { groupId = "org.apache.pekko", artifactId = "pekko-actor" },
   { groupId = "org.apache.pekko", artifactId = "pekko-actor-typed" },
```

Comment:
can you add the sbt paradox ignores? - eg https://github.com/apache/incubator-pekko/blob/main/.scala-steward.conf#L14

Reply:
Done and pushed.